### PR TITLE
Create Upload SQL files doc

### PR DESCRIPTION
**Pull request by alexanderahn**
Also makes minor updates to Use project tests doc to set the page's position.

### CODE REVIEW

File: docs/low-code-sql/development/upload-sql-files.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
```
Comment (atbida) [inline docs/low-code-sql/development/upload-sql-files.md:1]: I would have put this file as a subpage of this existing page: 
docs/low-code-sql/development/gems/datasources.md

If you want to do that, here are the steps:
1. create this path: `docs/low-code-sql/development/gems/datasources/`
2. move datasources.md and upload-sql-files.md into this ^ path.
3. create a new file `_category_.json` on this ^ path 
3a. set `"label": "datasources",` 
4. move the *.png files used in datasources.md and upload-sql-files into a new directory `docs/low-code-sql/development/gems/datasources/img`

**Review by atbida — APPROVED** [review]
LGTM!
If you prefer to move the file under `datasources` the instructions are in my comment. Ideally we don't want too many top-level docs, but perhaps this one is important enough that we don't want to bury it. Your call - looks great!

File: docs/low-code-sql/development/upload-sql-files.md (hunk 1 above)
Comment (alexanderahn) [reply docs/low-code-sql/development/upload-sql-files.md:1]: Done.

Had to set the label to Data Sources.

**Comment by alexanderahn** [comment]
Moved under datasources in https://github.com/SimpleDataLabsInc/prophecy-docs/pull/312

**Pull request merged**